### PR TITLE
Fixed #29536 -- Fixed SelectFilter2.js resizing to make boxes have equal height.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: django/contrib/admin/static/admin/js/SelectFilter2.js [hunk 1]
```diff
@@ -164,9 +164,11 @@ Requires jQuery, core.js, and SelectBox.js.
 
             if (!is_stacked) {
                 // In horizontal mode, give the same height to the two boxes.
-                var j_from_box = $(from_box);
-                var j_to_box = $(to_box);
-                var resize_filters = function() { j_to_box.height($(filter_p).outerHeight() + j_from_box.outerHeight()); };
+                var j_from_box = $('#' + field_id + '_from');
+                var j_to_box = $('#' + field_id + '_to');
+                var resize_filters = function() {
```
Comment (timgraham) [inline django/contrib/admin/static/admin/js/SelectFilter2.js]: I'd omit this unrelated cosmetic change.

**Comment by claudep** [comment]
I'm not sure either. But `from_box` is manipulated in various ways in the code above. I guess that at some point the object get cloned and what we have is no longer the real select element.
